R/create_query_data.R

```diff
@@ -280,38 +233,25 @@ create_query_data <- function(queries,
             QUERY_SCOPE_NUM = if_else(QUERY_SCOPE == "BROAD", 1, 2)
           )
       }
-    } else if (inherits(queries[[i]]$definition, "sdg_select")) {
-      # query is a SDG
-      query_data[[i]] <- get_terms_from_db(
-        version = whodd_version,
-        fun = get_sdg_fun,
-        queries = queries,
-        definition = queries[[i]]$definition,
-        expect_query_name = TRUE,
-        expect_query_id = !is.null(queries[[i]]$id),
-        i = i,
-        temp_env = temp_env,
-        type = "SDG"
-      )
     } else if (is.data.frame(queries[[i]]$definition)) {
       # query is a customized query
       query_data[[i]] <- queries[[i]]$definition
     } else if (is.list(queries[[i]]$definition)) {
-      # query is defined by customized queries and SMQs
+      # query is defined by customized queries and baskets
       definition <- queries[[i]]$definition
       terms <- vector("list", length(definition))
       for (j in seq_along(definition)) {
         if (is.data.frame(definition[[j]])) {
           terms[[j]] <- definition[[j]]
         } else {
           terms[[j]] <- get_terms_from_db(
-            version = meddra_version,
-            fun = get_smq_fun,
+            version = version,
+            fun = get_terms_fun,
             queries = queries,
             definition = definition[[j]],
             i = i,
             temp_env = temp_env,
-            type = "SMQ"
+            type = definition[[j]]$type
```

Comment:
This line should be removed as the `type` argument was removed from `get_terms_from_db()`.